lib/cc/engine/scalastyle.rb

```diff
@@ -57,22 +57,28 @@ def analysis_files
       end
 
       def scalastyle_jar
-        '/home/app/scalastyle_2.11-0.6.0-batch.jar'
+        '/home/app/scalastyle_2.11-0.8.0-batch.jar'
       end
 
       def scalastyle_config
-        @engine_config['config'] || '/home/app/scalastyle_config.xml'
+        @engine_config['config'] || '/code/scalastyle_config.xml'
       end
 
       def results_xml_file_path
         "/home/app/results.xml"
       end
 
+      def results_xml_tmp_file_path
+        "/home/app/results.tmp.xml"
+      end
+
       def exec_scalastyle_with_options
+        Dir.chdir "/code"
         `java -jar #{scalastyle_jar} \
-          -c #{scalastyle_config} \
-          --xmlOutput #{results_xml_file_path} \
+          --config #{scalastyle_config} \
+          --xmlOutput #{results_xml_tmp_file_path} \
           #{analysis_files}`
+        `cat #{results_xml_tmp_file_path} | sed -e 's/file name=\"\\\/code\\\//file name=\"/g' > #{results_xml_file_path}`
```

Comment:
So I was going to suggest handling that here: https://github.com/codeclimate-community/codeclimate-scalastyle/blob/master/lib/cc/engine/scalastyle.rb#L19 rather than `sed`ing the raw XML -- but now that I'm looking at that code, is that a bug? Shouldn't that line be `sub`bing out `/^\/code\//`?